sim/deploy/config.py

```diff
@@ -57,7 +57,7 @@ class RobotConfig:
     kds: np.ndarray
     tau_limit: np.ndarray
     robot_model_path: str
-    dt: float = 0.001
+    dt: float = 0.0001  # 0.001
```

Comment:
why do we change this?

Reply:
It slows the video down so we can watch it at a reasonable speed